data/card_catalogue/movement.json

```diff
@@ -57,7 +69,10 @@
     "upgrade": "",
     "tier": "green",
     "details": {
-      "rules": "Draws you towards a charcter in your line of sight at high speed. Both take 6 damage on impact.",
+      "rules": {
+        "active": "Pick a visible character to be flung towards. Both take 6 damage on impact.",
```

Comment:
made this one a bit more concise since I think it's a good candidate for:
- #106